modules/graphman-utils.js

```diff
@@ -156,7 +156,7 @@ module.exports = {
 
     writeFileBinary: function (file, data) {
         this.mkDir(this.parentPath(file));
-        fs.writeFileSync(file, data);
+        fs.writeFileSync(file, data, { encoding: 'binary' });
```

Comment:
there's no **encoding** as **binary**.

https://nodejs.org/docs/v20.12.1/api/fs.html#fsreadfilesyncpath-options

Reply:
Assuming that, the data is buffer, we can ignore this line of change as well.